application/client/src/app/service/session/dependencies/search/charts/store.ts

```diff
@@ -1,13 +1,22 @@
 import { Key, Store, StoredEntity } from '../store';
 import { ChartRequest } from './request';
 import { IFilter } from '@platform/types/filter';
-import { DisableConvertable } from '../disabled/converting';
-
+import { Subject } from '@platform/env/subscription';
+import { ISelectEvent } from '@ui/views/sidebar/search/providers/definitions/provider';
 export { ChartRequest } from './request';
 
 export class ChartsStore extends Store<ChartRequest> {
+    public chartSelected: Subject<string> = new Subject<string>();
```

Comment:
this is not an event of `Store` and entity. This is UI event. It should not "lives" here